src/app/workbench/WorkbenchViews.tsx

```diff
@@ -1,5 +1,24 @@
 import * as React from 'react';
-import SplitPane from 'react-split-pane';
+import {
+  Corner,
+  createRemoveUpdate,
+  getLeaves,
+  getNodeAtPath,
+  getOtherDirection,
+  getPathToCorner,
+  Mosaic,
+  MosaicDirection,
+  MosaicNode,
+  MosaicParent,
+  MosaicPath,
+  updateTree,
+} from 'react-mosaic-component';
+import 'react-mosaic-component/react-mosaic-component.css';
+
+import '@blueprintjs/core/lib/css/blueprint.css';
+import '@blueprintjs/icons/lib/css/blueprint-icons.css';
```

Comment:
@puehringer Should we keep the CSS imports here or move them to a SCSS file as we usually do?

Reply:
Mmh, I am actually torn on what we should for "static" assets. In general, this method has some benefits (code splitting and lazy loading if possible), however it opens up the issue where people might want to use mosaic (as it is in the npm#dependencies), but they need to remember to import the css for it. If we add it to `vendors/mosaic.scss`, it would be part of the "big" css and globally available.

Reply:
@puehringer Thanks for your opinion. Then I'd suggest to follow our convention and create a vendors SCSS file with the CSS imports. If we have better code splitting in place, we can refactor the SCSS files and imports anyway. Following the convention would make it easier for the developer doing the refactoring.

@dvzacharycutler Please create a `src/scss/vendors/_react-mosaic.scss` file and move the CSS imports to there. It will look similar to the [font-awesome.scss in tdp_core](https://github.com/datavisyn/tdp_core/blob/master/src/scss/vendors/_font-awesome.scss). Afterwards import the vendors file in the main.scss, similar to https://github.com/datavisyn/tdp_core/blob/3ce118d51eb023e1fa5ea708ff3dbf0f3662a870/src/scss/main.scss#L14. Thanks.